src/main/java/the_warlord/powers/BleedPower.java

```diff
@@ -0,0 +1,73 @@
+package the_warlord.powers;
+
+import basemod.interfaces.CloneablePowerInterface;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.MathUtils;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePrefixPatch;
+import com.megacrit.cardcrawl.actions.common.LoseHPAction;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.powers.watcher.VigorPower;
+import the_warlord.WarlordMod;
+import the_warlord.characters.Warlord;
+
+import java.util.Random;
+
+public class BleedPower extends AbstractPower implements CloneablePowerInterface {
+    public static final String POWER_ID = WarlordMod.makeID("BleedPower");
+    private static final PowerStrings powerStrings = CardCrawlGame.languagePack.getPowerStrings(POWER_ID);
+    public static final String NAME = powerStrings.NAME;
+    public static final String[] DESCRIPTIONS = powerStrings.DESCRIPTIONS;
+
+    public BleedPower(AbstractCreature owner, int amount) {
+        name = NAME;
+        ID = POWER_ID;
+        this.owner = owner;
+        this.amount = amount;
+        type = PowerType.BUFF;
+        updateDescription();
+        isTurnBased = false;
+        // TODO: add proper art
+        loadRegion("explosive");
+    }
+
+    @Override
+    public void updateDescription() {
+        this.description = DESCRIPTIONS[0];
+        /*
+        if (this.owner.hasPower(GashPower.POWER_ID)) { description = "plc_gash_1"; }
+        else { description = "plc_nogash_1"; }
+
+         */
+        }
+
+    @Override
+    public void atStartOfTurn() {
+        if (this.owner.hasPower(GashPower.POWER_ID)) {
+            AbstractDungeon.actionManager.addToBottom(new LoseHPAction(this.owner, this.owner, this.amount));
+            AbstractDungeon.actionManager.addToBottom(new ReducePowerAction(this.owner, this.owner, this, this.amount));
+        }
+    }
+
+    @Override
+    public void stackPower(int i) {
+        super.stackPower(i);
+        updateDescription();
+    }
```

Comment:
I don't see a reason to override this method. Stacking behaviour should be managed by CustomWarlordModPower. Unless there's something specific to BleedPower that I'm missing.